lib/broccoli/ember-app.js

```diff
@@ -645,20 +649,16 @@ class EmberApp {
   */
   index() {
     let htmlName = this.options.outputPaths.app.html;
-    let files = [
-      'index.html',
-    ];
 
-    let index = new Funnel(this.trees.app, {
-      files,
-      getDestinationPath(relativePath) {
-        if (relativePath === 'index.html') {
-          relativePath = htmlName;
-        }
-        return relativePath;
-      },
-      annotation: 'Funnel: index.html',
-    });
+    let index;
+    if (!this.trees.src) {
+      index = this._rawAppIndex(htmlName);
+
```

Comment:
yeah, that's somewhat deliberate, but I can remove it if you're feeling annoyed by it ;)